_layouts/global.html

```diff
@@ -211,10 +216,12 @@ <h5>Latest News</h5>
 
 <footer class="small">
   <hr>
-  Apache Spark, Spark, Apache, and the Spark logo are <a href="{{site.baseurl}}/trademarks.html">trademarks</a> of
-  <a href="https://www.apache.org">The Apache Software Foundation</a>.
+  Apache Spark, Spark, Apache, the Apache feather logo, and the Apache Spark project logo are either registered
```

Comment:
This new text is pulled from the trademarks page and should make https://whimsy.apache.org/site/project/spark happier.